app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -121,6 +126,12 @@ class ToolDetailsFragment() : BasePlatformFragment<ToolDetailsFragmentBinding>(R
             }
         }
     }
+
+    fun openTraining(manifest: Manifest) {
+        if (manifest.type == Manifest.Type.TRACT) {
```

Comment:
hmm, I hadn't realized we had a different manifest & tool type

Reply:
Yeah, just Article and Tract.  Is it ok to leave it like this or should I handle the else condition

Reply:
this is fine, at some point I'll re-evaluate the types to see if there is a better option